src/components/Profile/ProfileHeader.tsx

```diff
@@ -14,7 +14,8 @@ export function ProfileHeader({ profileId }: { profileId: string }) {
   const session = useSession();
   const friendsList = user.data?.friendsA.concat(user.data?.friendsB);
 
-  session.data?.user?.id === user.data?.id;
+  const profilePhotoUrl = `${process.env.NEXT_PUBLIC_SUPABASE_URL}/storage/v1/object/public/images/${user.data?.profilePhotoUrl}`;
```

Comment:
its best to use the exported env from the env.ts file so you get the autocomplete on environment variables here